src/readDeviceCapabilities.ts

```diff
@@ -48,5 +48,7 @@ export async function readDeviceCapabilities(): Promise<DeviceCapabilities> {
     platenMaxHeight: scanCaps?.platenMaxHeight || null,
     adfMaxWidth: scanCaps?.adfMaxWidth || null,
     adfMaxHeight: scanCaps?.adfMaxHeight || null,
+    adfDuplexMaxWidth: scanCaps?.adfDuplexMaxWidth || null,
```

Comment:
@aaliddell It appears that this small change in the file has increased the complexity of this function beyond CodeFactor's default threshold. Could you look into simplifying this modification? Thank you!